rwengine/src/objects/VehicleObject.hpp

```diff
@@ -36,6 +37,7 @@ class VehicleObject final : public GameObject, public ClumpObject {
     float throttle{0.f};
     float brake{0.f};
     bool handbrake = true;
+    btScalar *wheelsRotation;
```

Comment:
It asks for `std::unique_ptr`.

Reply:
Rather ``std::vector``

Reply:
Good point. ;)

Reply:
Thanks, just pushed an updated version with std::vector.